### ISSUE TITLE
dataclasses issues

### ISSUE THREAD

**Issue by good-coder-bad-boy** [rr-/screeninfo#41]
I have a feeling it has to do with the `@dataclass` tag above the `Monitor` class in `common.py`:
```
Traceback (most recent call last):
    File "test.py", line 1 in <module>
        from screeninfo import get_monitors
    File "python3/lib/python3.8/site-packages/screeinfo/__init__.py", line 1, in <module>
        from .common import Enumerator, Monitor
    File "python3/lib/python3.8/site-packages/screeinfo/common.py", line 7, in <module>
        class Monitor:
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 958, in dataclass
        return wrap(_cls)
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 950, in wrap
        return _process_class(cls, init, repr, eq, order, unsafe_hash, frozen)
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 800, in _process_class
        cls_fields = [_get_field(cls, name, type)
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 800, in <listcomp>
        cls_fields = [_get_field(cls, name, type)
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 659, in _get_field
        if (_is_classvar(a_type, typing)
    File "python3/lib/python3.8/site-packages/dataclasses.py", line 659, in _is_classvar
        return (a_type is typing._ClassVar
AttributeError: module 'typing' has no attribute '_ClassVar'
```
For some reason my editor seems to have placed an underscore in front of some of the variables. I have looked at the [source code](https://github.com/python/cpython/blob/3.8/Lib/dataclasses.py) and they are not there

**Comment by rr-** [comment]
Is this the same issue as with https://github.com/google/flax/issues/269 ?

**Comment by good-coder-bad-boy** [comment]
yes

**Comment by rr-** [comment]
Fixed in version 0.6.6
